lib/model.js

```diff
@@ -121,14 +192,14 @@ module.exports = class Model {
 			let type = attribute && exists(attribute.type) && attribute.type;
 			let transform = attribute && isFunction(attribute.transform) && attribute.transform;
 
-			let value = values && exists(values[key]) ? values[key] : null;
+			let value = values && exists(values[key]) ? values[key] : undefined;
 			value = ( type !== 'function' && isFunction(value) ) ? value(key, type) : value;
```

Comment:
just checking this is the intended logic - type is not function so we go ahead and invoke it as a function?